components/emoji_picker/emoji_picker.jsx

```diff
@@ -217,6 +235,23 @@ export default class EmojiPicker extends React.PureComponent {
         }
     }
 
+    componentWillUnmount() {
+        if (this.renderAllCategoriesFrame) {
+            window.cancelAnimationFrame(this.renderAllCategoriesFrame);
```

Comment:
what about the parent animation frame? or is it too fast that we don't need to consider it? 0/5 if we cancel it here

Reply:
Ya, It is too fast so was not good enough